pkgs/tools/X11/xpointerbarrier/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, xorg, fetchurl }:
+stdenv.mkDerivation rec {
+  name = "xpointerbarrier-${version}";
+  version = "17.10";
+
+  src = fetchurl {
+    url = "https://github.com/vain/xpointerbarrier/archive/v${version}.tar.gz";
+    sha256 = "0jh9kn7yrc5khfwdrrciw7bf6bizyriw74h3a473gnbgnrcknm58";
+  };
+
+  buildInputs = [ xorg.libX11 xorg.libXfixes xorg.libXrandr ];
+
+  makeFlags = "prefix=$(out)";
```

Comment:
This should be the default. Does it fail to build without this line?

Reply:
Yes it fails. I guess default have uppercase `PREFIX`.

Reply:
Ah, just looked at the code and this doesn't use a configure script, makes sense now.